plugins/dfu/dfu.quirk

```diff
@@ -487,3 +487,16 @@ DfuForceVersion = 0x0
 DfuForceVersion = 0x0
 [USB\VID_10A5&PID_9800]
 DfuForceVersion = 0x0
+
+# HID OMNIKEY card readers 
+[USB\VID_076B] 
+Vendor = HID OMNIKEY 
+[USB\VID_076B&PID_5127] 
+Plugin = dfu 
```

Comment:
Do you need this line? Dfu devices are normally autodetected by the interface data. Also precommit days you have trailing spaces -- if you ammend the commit and force push to your branch it'll update this PR.

Reply:
Line 495? I wasn't sure. Looking at other entries in this file I so both, devices with and without "Plugin - dfu". If that's redundant, I don't need it. 
As for trailing spaces - I don't see those in my editor. Or is this a reference to the spaces in Vendor and Product entries? Those are strings that are also generated this way during the enumeration of the USB device.

Reply:
Is the device detected correctly without the Plugin = dfu line?

> As for trailing spaces - I don't see those in my editor

Are you editing in Windows? i.e. have you got win32 line endings perhaps?

Reply:
Yes, the file was edited on a Windows system. It is possible that a paste from the clipboard messed up the line endings. I'll check and correct it if that's what happened.  I accessed GitHub & the forked repo directly using a Chrome browser to prevent this type of issue. 

(pending) - I checked with the test lab about their requirements regarding  Plugin = dfu

Reply:
I tested this with an OMNIKEY 5427 CK and don't see the device when running `fwupdmgr get-devices` with or without the `Plugin = dfu` lines.

When running `fwupd` with the `Plugin = dfu` lines I got the following traces:
```
06:43:01.721 FuEngine             usb:03:00:02:04:01:02 added FuUsbDevice:
  Name:                 5427 CK
  Guid:                 b08121dc-c4b7-5085-940e-52ccadbd30de ← USB\VID_076B&PID_5427 ⚠
  Guid:                 9b539786-e2bc-506f-86f3-b12f1c2ae7eb ← USB\VID_076B&PID_5427&REV_0100 ⚠
  Flags:                none
  Vendor:               HID OMNIKEY
  VendorId:             USB:0x076B
  Version:              1.0
  VersionFormat:        bcd
  VersionRaw:           0x00000100
  Guid[quirks]:         4777d075-986d-5f97-a088-d46c8851e5c4 ← USB\VID_076B
  Guid[quirks]:         72ab5b5f-1658-58ac-b117-abf14a1724f8 ← USB\CLASS_0B
  Guid[quirks]:         ac72c389-982a-519b-87eb-ecdb411f9547 ← USB\CLASS_0B&SUBCLASS_00
  Guid[quirks]:         b1613457-7705-543f-869b-668865fc5176 ← USB\CLASS_0B&SUBCLASS_00&PROT_00
  Guid[quirks]:         9da74da6-bfa4-53cc-8cc1-7559a7419a15 ← USB\CLASS_02
  Guid[quirks]:         58abff3a-7763-585f-baa7-e71e45dfa1fd ← USB\CLASS_02&SUBCLASS_0C
  Guid[quirks]:         7dbfcac9-afaf-581b-8dde-d19336cee6db ← USB\CLASS_02&SUBCLASS_0C&PROT_07
  PhysicalId:           usb:03:00:02:04:01:02
  AcquiesceDelay:       2500
  PossiblePlugin:       dfu
  ParentPhysicalIds:    usb:03:00:02:04:01,usb:03:00:02:04
  UsbDeviceClass:       interface-desc

06:43:01.723 FuEngine             dfu ignoring: 076b:5427 is not supported: no DFU interfaces
```
And without the `Plugin = dfu` lines:
```
06:56:12.613 FuEngine             usb:03:00:02:04:01:02 added FuUsbDevice:
  Name:                 5427 CK
  Guid:                 b08121dc-c4b7-5085-940e-52ccadbd30de ← USB\VID_076B&PID_5427 ⚠
  Guid:                 9b539786-e2bc-506f-86f3-b12f1c2ae7eb ← USB\VID_076B&PID_5427&REV_0100 ⚠
  Flags:                none
  Vendor:               HID OMNIKEY
  VendorId:             USB:0x076B
  Version:              1.0
  VersionFormat:        bcd
  VersionRaw:           0x00000100
  Guid[quirks]:         4777d075-986d-5f97-a088-d46c8851e5c4 ← USB\VID_076B
  Guid[quirks]:         72ab5b5f-1658-58ac-b117-abf14a1724f8 ← USB\CLASS_0B
  Guid[quirks]:         ac72c389-982a-519b-87eb-ecdb411f9547 ← USB\CLASS_0B&SUBCLASS_00
  Guid[quirks]:         b1613457-7705-543f-869b-668865fc5176 ← USB\CLASS_0B&SUBCLASS_00&PROT_00
  Guid[quirks]:         9da74da6-bfa4-53cc-8cc1-7559a7419a15 ← USB\CLASS_02
  Guid[quirks]:         58abff3a-7763-585f-baa7-e71e45dfa1fd ← USB\CLASS_02&SUBCLASS_0C
  Guid[quirks]:         7dbfcac9-afaf-581b-8dde-d19336cee6db ← USB\CLASS_02&SUBCLASS_0C&PROT_07
  PhysicalId:           usb:03:00:02:04:01:02
  AcquiesceDelay:       2500
  ParentPhysicalIds:    usb:03:00:02:04:01,usb:03:00:02:04
  UsbDeviceClass:       interface-desc
```